utils.py

```diff
@@ -200,12 +200,14 @@ def load_checkpoint(coarse_model, fine_model, optimizer, args):
         # Load model
         coarse_model.load_state_dict(ckpt['coarse_model_state_dict'], strict=False)
         for param in coarse_model.parameters():
-            param.requires_grad = False
+            param.requires_grad = True
+            # param.requires_grad = False
```

Comment:
I would just delete this line rather than leave it commented -- or maybe pass `requires_grad` as an argument to the load function?

Reply:
Agreed. See latest push for this change (and the to8b comment above)